azure-pipelines/generate-target-database.yaml

```diff
@@ -30,7 +30,13 @@ jobs:
         inputs:
           versionSpec: '3.7'
 
-      - template: steps/install-development-dependencies.yml
```

Comment:
Can we delete the file `steps/install-development-dependencies.yml`?

Reply:
Not yet, it's still used in the `build-and-release` pipeline. When we move to `tox` we won't need it anymore.